src/node_contextify.h

```diff
@@ -41,34 +41,37 @@ struct ContextOptions {
   BaseObjectPtr<MicrotaskQueueWrap> microtask_queue_wrap;
 };
 
-class ContextifyContext {
+class ContextifyContext : public BaseObject {
  public:
   ContextifyContext(Environment* env,
-                    v8::Local<v8::Object> sandbox_obj,
+                    v8::Local<v8::Object> wrapper,
+                    v8::Local<v8::Context> v8_context,
                     const ContextOptions& options);
   ~ContextifyContext();
-  static void CleanupHook(void* arg);
 
+  void MemoryInfo(MemoryTracker* tracker) const override;
+  SET_MEMORY_INFO_NAME(ContextifyContext)
+  SET_SELF_SIZE(ContextifyContext)
+
+  static BaseObjectPtr<ContextifyContext> New(Environment* env,
+                                              v8::Local<v8::Object> sandbox_obj,
+                                              const ContextOptions& options);
   static v8::MaybeLocal<v8::Context> CreateV8Context(
       v8::Isolate* isolate,
       v8::Local<v8::ObjectTemplate> object_template,
       const SnapshotData* snapshot_data,
       v8::MicrotaskQueue* queue);
-  bool InitializeContext(v8::Local<v8::Context> ctx,
-                         Environment* env,
-                         v8::Local<v8::Object> sandbox_obj,
-                         const ContextOptions& options);
+  static bool InitializeContext(v8::Local<v8::Context> ctx,
```

Comment:
It looks to me that this method has been merged into the `ContextifyContext::New`. Do we still need this? Or can we leave this as a separate method but mark it as a private method as `ContextifyContext::New` is growing relatively large?

Reply:
Oh, right, it's probably still looks a bit nicer to keep this as a private method instead of merging it. Thanks for catching it!